sql/2020/20_Caching/cache_ttl_lighthouse_score.sql

```diff
@@ -0,0 +1,15 @@
+#standardSQL
+# Distribution of cache TTL Lighthouse scores
+SELECT 
+  _TABLE_SUFFIX AS client,
+  JSON_EXTRACT_SCALAR(report, "$.audits.uses-long-cache-ttl.score") AS caching_score,
+  COUNT(*) AS num_pages,
+  COUNT(0) * 100 / SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS pct_pages

```

Comment:
A few nits:

- `COUNT(0)` consistently rather than `COUNT(*)`
- always show your work by including the denominator AKA "total" field, sometimes this is helpful to spot mistakes
- don't `* 100` in SQL, let Sheets format numbers as percents
```suggestion
  COUNT(0) AS num_pages,
  SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS  total,
  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS pct_pages
```